modules/billing-exports/README.md

```diff
@@ -0,0 +1,198 @@
+<!-- Update the title -->
+# billing-exports module
```

Comment:
Use a more human readable title here (the actual module title is already below on line 70)